### PR TITLE
Fix Traffic Ops CDN DNSSEC generation when no delivery services exist

### PR DESCRIPTION
**Pull request by rob05c**
Fixes an edge case of generating DNSSEC keys on a CDN when no delivery
services exist on that CDN (for example, if generating immediately
after creating the CDN). The edge case caused keys to be generated
with no domain name.

API Tests don't support Riak yet.

Manually tested, verified DNSSEC generation on a new CDN with no DSes works, and generates keys with the CDN's domain. Tested that DNSSEC generation and regeneration on a CDN with DSes still works.

#### What does this PR do?

Fixes #(issue_number)

#### Which TC components are affected by this PR?

- [ ] Documentation
- [ ] Grove
- [ ] Traffic Analytics
- [ ] Traffic Monitor
- [x] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] Other _________

#### What is the best way to verify this PR?

Create a new CDN and generate DNSSEC keys. Verify the keys at `/api/1.2/cdns/name/your-new-cdn-name/dnsseckeys` have `name` of the cdn's domain name, and not `.`.

#### Check all that apply

- [ ] This PR includes tests
- [ ] This PR includes documentation updates
- [ ] This PR includes an update to CHANGELOG.md
- [ ] This PR includes all required license headers
- [ ] This PR includes a database migration (ensure that migration sequence is correct)
- [ ] This PR fixes a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)

### CODE REVIEW

File: traffic_ops/traffic_ops_golang/cdn/dnssec.go [hunk 1]
```diff
@@ -245,7 +257,7 @@ type CDNDS struct {
 }
 
 // getCDNDeliveryServices returns basic data for the delivery services on the given CDN, as well as the CDN name, or any error.
-func getCDNDeliveryServices(tx *sql.Tx, cdn string) ([]CDNDS, string, error) {
+func GetCDNDeliveryServices(tx *sql.Tx, cdn string) ([]CDNDS, error) {
 	q := `
 SELECT ds.xml_id, ds.protocol, t.name as type, ds.routing_name, cdn.domain_name as cdn_domain
```
Comment (rawlinp) [inline traffic_ops/traffic_ops_golang/cdn/dnssec.go]: The `cdn.domain_name as cdn_domain` should be removed from the query if it's not being scanned out from the result, right?

Comment (rob05c) [reply traffic_ops/traffic_ops_golang/cdn/dnssec.go]: Yep. Fixed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3145/
Test PASSed.
